GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationSectorController.cs

```diff
@@ -128,6 +183,48 @@ private void AddNewOrganisationPublicSectorType(Organisation organisation, int p
 
             organisation.LatestPublicSectorType = newOrganisationPublicSectorType;
         }
+        
+        private void UpdateAdminChangeSectorViewModelFromOrganisation(AdminChangeSectorViewModel viewModel, long organisationId)
+        {
+            viewModel.Organisation = dataRepository.Get<Organisation>(organisationId);
+
+            viewModel.InactiveUserOrganisations = dataRepository.GetAll<InactiveUserOrganisation>()
+                .Where(m => m.OrganisationId == organisationId).ToList();
+        }
+        
+        private void ValidateAdminChangeSectorViewModel(AdminChangeSectorViewModel viewModel)
+        {
+            if (!viewModel.NewSector.HasValue)
+            {
+                viewModel.AddErrorFor(m => m.NewSector, "Please select a new sector");
+            }

```

Comment:
Instead of this, I think we could use a `GovUkValidateRequired` attribute on the view-model